context/app/static/js/components/detailPage/MetadataSection/MetadataSection.tsx

```diff
@@ -55,6 +55,7 @@ function MetadataWrapper({ allTableRows, tsvColumns = defaultTSVColumns, childre
         <SecondaryBackgroundTooltip title="Download">
           <a href={downloadUrl} download={`${hubmap_id}.tsv`} aria-label="Download TSV of selected items' metadata">
             <StyledWhiteBackgroundIconButton
+              aria-label="Download TSV of selected items' metadata"
```

Comment:
Generally, it's best to avoid directly duplicating a label since it would mean the same text gets read twice when the screenreader focuses the item.

This is an interesting case since it's a button inside of a link; after doing some research, it seems like nesting buttons inside of `a` tags is a discouraged pattern for accessibility reasons, but it should apparently be possible to make it work without duplicating the aria label by using `aria-role`:

```
<a href={downloadUrl} download={`${hubmap_id}.tsv`}> // remove the label from the parent link
  <StyledWhiteBackgroundIconButton
    aria-label="Download TSV of selected items' metadata"
    aria-role="link"
```

Does this approach resolve the issue in Axe?

Reply:
That fixed the issue, thanks for the suggestion. The role for a button is inherently `button` by default, so I did not add it.